### PR TITLE
Subport branch

### PR DESCRIPTION
**Pull request by Daphineatim**
* Added about section
*  Added projects section

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -145,3 +144,147 @@ ul.social-media-icons {
...
   width: 100vw;
 }
+
+/* Works */
+#works {
+  margin-top: 100px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
```
Comment (micheaol) [inline style.css]: Please ensure that the main container of the `projects section` has a `display: grid` according to the requirement.

File: index.html [hunk 2]
```diff
@@ -1,79 +1,226 @@
...
+          <h1 class="about__title">
+            About <span>Myself</span>
+         </h1>
+         <p class="about__dsc">
+            Hello I'm a software developer! I can help you build a product , feature or
+            website Look through some of my work and experience! If you like what
+            you see and have a project you need coded, don’t hestiate to contact
+            me.
+         </p>
+         <a href="#" class="head__link">LET'S CONNECT</a>
```
Comment (micheaol) [inline index.html]: Please ensure that the titles (e.g., "Let's connect") use a header tag `hx` e,g `h1, h2, h3`

File: index.html [hunk 3]
```diff
@@ -1,79 +1,226 @@
...
+             <div class="accordion_section_title no-border">
+               Languages 
+               <img src="/assets/chev-down.png" class="chev pull-right" alt="image">
+             </div>
+             <div class="accordion_section_body">
+              <div class="languages_list">
+                <div class="language"><img src="/assets/javascript.svg.svg" class="language_img" alt="icon"><span>Javascript</span></div>
+                <div class="language"><img src="/assets/html.svg.svg" class="language_img" alt="icon"><span>HTML</span></div>
+                <div class="language"><img src="/assets/css.svg.svg" class="language_img" alt="icon"><span>CSS</span></div>
+              </div>
```
Comment (micheaol) [inline index.html]: Please ensure that the technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li` and the technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li`

File: index.html [hunk 4]
```diff
@@ -1,79 +1,226 @@
...
+         <div class="card_works">
+           <img class="card_works_image" src="./assets/multi_post_2.svg" alt="image">
+           <h2 class="card_works_heading">Multi-Post Stories</h2>
+           <div class="card_works_links">
+             <span>Canopy</span>
+             &middot;
+             <span>Backend Dev</span>               
+             &middot;
+             <span>2015</span>
+          </div>
```
Comment (micheaol) [inline index.html]: Please ensure that you stick to the design as much as possible (e.g., font, colors, images, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). Please ensure that you this adjustment to all the work cards.

Your Implementation | Figma Design
:---:|:---:
image | image

File: index.html [hunk 5]
```diff
@@ -1,79 +1,226 @@
...
+            <button class="card_works_button">See Project</button>
+         </div>
+         <div class="card_works">
+           <img class="card_works_image" src="./assets/tonic_2.svg" alt="image">
+           <h2 class="card_works_heading">Tonic</h2>
+           <div class="card_works_links">
+             <span>Canopy</span>
+             &middot;
+             <span>Backend Dev</span>               
+             &middot;
```
Comment (micheaol) [inline index.html]: Please ensure that the work section has the margins (top, bottom, left, right) between each card as defined in the design. Also please ensure that the button is as defined in the design.

Your Implementation | Figma Design
:---:|:---:
image | image

File: index.html [hunk 6]
```diff
@@ -1,79 +1,226 @@
...
+          <section class="about flex">
+          <h1 class="about__title">
+            About <span>Myself</span>
+         </h1>
+         <p class="about__dsc">
+            Hello I'm a software developer! I can help you build a product , feature or
+            website Look through some of my work and experience! If you like what
+            you see and have a project you need coded, don’t hestiate to contact
+            me.
+         </p>
```
Comment (micheaol) [inline index.html]: Please ensure that you stick to the design as much as possible (e.g., font, colors, images, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). Please ensure that the `paragraph` has 6 lines as in the design.

Your Implementation | Figma Design
:---:|:---:
image | image

File: index.html [hunk 7]
```diff
@@ -1,79 +1,226 @@
...
+               <li class="li">
+                  <a href="https://medium.com/@hopedaphine19" class="assets link"
+                     ><img src="./assets/medium.svg.svg" alt="medium icon"
+                     /></a>
+               </li>
+               <li class="li">
+                  <a href="https://github.com/Daphineatim" class="assets link"
+                     ><img src="./assets/github.svg.svg" alt="github icon"
+                     /></a>
+
```
Comment (micheaol) [inline index.html:184]: Please ensure that you stick to the design as much as possible (e.g., font, colors, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Your Implementation | Figma Design
:---:|:---:
image | image

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @Daphineatim , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 8]
```diff
@@ -1,79 +1,323 @@
...
+               I'm Daphine <span>Glad to see you!</span>
+            </h1>
+            <p class="head__dsc fz-16 fw-400 1h-24">
+               I'm a software developer! I can help you build a product , feature or
+               website Look through some of my work and experience! If you like what
+               you see and have a project you need coded, don’t hestiate to contact
+               me.
+            </p>
+            <a href="#" class="head__link">LET'S CONNECT</a>
+            <div class="assets__icons">
```
Comment (Donard97) [inline index.html]: - [x] Kindly change link LET'S CONNECT to a header tag hx e,g h2,h3,h4 like the other reviewer requested you to do. Keep rocking 💪

**Review by Donard97 — CHANGES_REQUESTED** [review]
## Hi @Daphineatim , 👋  
    
Great work on making the changes requested by a previous reviewer 👏 

You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

giphy (5)

## HIGHLIGHTS ✍️ 

- Your project is professional ✔️ 
- Files are structured well ✔️ 
- There is no linter error ✔️ 
- Well done with UI design ✔️ 

## Required Changes ♻️

- [ ] Please check the comments under the review.
    
## Optional suggestions
    
 _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [OPTIONAL] I suggest for you to add a screenshot of you project on README file to make it perfect!
    
Cheers and Happy coding!👏👏👏
    
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.
    
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
    
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
````

**Review by Donard97 — APPROVED** [review]
## Hi @Daphineatim , 👋  

**Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:** 


## Congratulations! 🎉 🎉 🎉 

giphy (6)

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**